clog/context.go

```diff
@@ -42,6 +42,8 @@ const (
 	contextServerID
 	contextSchedulerType
 	contextCloudType
+	contextCallValue
+	contextServerFlavour
```

Comment:
Can we stick to the American spelling for Flavor throughout, since that is what OpenStack and other wr code calls it?